documentation.md

```diff
@@ -1,22 +1,68 @@
-| [ulid]({% link extensions/ulid.md %})                   | [<span class=github>GitHub</span>](https://github.com/Maxxen/duckdb_ulid)                          | ULID data type for DuckDB                                                                                                 |
+## Submit a Community Extension
 
+Extensions code are hosted as Github repositories.
+
+Users are encouraged to check the relevant extension repository for giving feedback, look at the implementation, and report issues.
+
+Repositories needs to be Public, Open Source, and hosted on GitHub.
+
+Submitting a Community Extension is a matter of opening a PR to the [Community Extensions Repository](https://github.com/duckdb/community-extensions) adding a single `descriptor.yml` file in the folder `extensions/$name_of_extension`.
+
+### YAML descriptor
+
+Structure of the YAML descritor is as follow:
+
+| field | description |
+|--|--|
+| extension.name | Name of the extension. Extension names are case-insensitive, so only lowercase letters, numbers and `-` or `_` are allowed |
+| extension.description |Short description of the extension | 
+| extension.version | Semantic versioning of the extension |
+| extension.language | `C++`, `Rust & C++`, `C++ & SQL`, or whatever combination of languages the extension code relies on |
+| extension.build | Build system the extensin uses, at the moment only `cmake` is supported |
+| extension.licence | Licence that applies to the extension |
+| extension.maintainers | A list of maintainers for the extension |
+| extension.excluded_platforms | Optional, allows to specify whether certain platforms are not supported |
+| extension.requires_toolchains | Optional, allows to specify whether additional build-time dependecies are required |
+| repo.github | Organization name '/' Repository name  of the GitHub-hosted public repository |
+| repo.ref | Git ref for the extension |
+
+These fields are used to build, test and deploy the extension, and might also be reflected in the documentation.
+
+Extensions maintainers are encouraged to provide two optional fields in the descriptor, to be only used in the autogenerated documentation:
+
+| field | description |
+|--|--|
+| docs.hello_world | An `Hello, World!`, a self contained example of an extension capabilities |
+| docs.extended_description | Extra context on the extension, relevant links, or a tour of the extension capabilities |
+
+### Hosted Extension Documentation Page 
+
+Each Community Extension has a documentation page at: `https://community-extension.duckdb.org/extensions/$extension_name.html`.
+
+For example, this is the page for [quack]({% link extensions/quack.md %}).
+
+Documentation pages are generated from the fields provided in the YAML descriptor file that is part of the [Community Extensions Repository](https://github.com/duckdb/community-extensions) and from the auto-detected changes that a given extensions introduces in DuckDB.
+
+#### Autodetected changes
```

Comment:
I think this header is not necessary? If you want to keep it i would suggest renaming it to 'documentation generation' and start with a sentence explaining that the section is going to explain how the documentation is generated

Reply:
Thanks!